articles/blablaconf-content-selection/index.md

```diff
@@ -0,0 +1,118 @@
+
+Having organized the [BlablaConf](https://blablaconf.com) conference for five years now, we have developed a process that we believe works quite well.
+
+As transparency is one of our core values, this article serves to share our process with you. It will provide answers for those interested in submitting a talk in the future, as well as insights into why your talk may not have been selected in the past.
+
+This article only covers the content selection process for BlablaConf, if you are interested in other aspects of the conference feel free to mention it in the comments section and we will cover it in another article.
+
+## 🇲🇦 What is BlablaConf?
+
+For those who don't know, [BlablaConf](https://blablaconf.com) is our annual Geeksblabla community conference entirely in Moroccan Darija. It's a 5+1 days conference where speakers from various backgrounds and levels of expertise share their knowledge and experience with the community. Mostly online with few in-person days around Morocco.
+
+## 📝 Call for papers
+
+After we agree on the date of the conference, typically in the first week of February, we open the call for papers three months prior to the event. The call for papers usually remains open for one and a half months.
+
+If you are not familiar with call for papers (CFP) term, it simply refers to a form that people need to fill out in order be part of the conference as speakers.
+
+We use [sessionize.com](https://sessionize.com/), a platform that helps us collect and manage submissions, making the selection process easier. We want to thank them for their excellent service, as it provides great assistance to us at no cost.
+
+In the call for papers form, we ask for the following information:
+
+- Speaker details
+- Talk title: The title of the talk
+- Talk description: This section outlines the content of your talk. It is the most important part of the form, so please invest time in writing it.
+- Talk level (beginner, intermediate, advanced)
+- Talk track: every day we have a different track (`web, Mobile & UX`, `Big data and machine learning`, `Backend & Programming Languages`, `Cloud, Containers & Infrastructure`, `Security & Architecture` ,`Soft Skills & Career`)
+- Talk type: keynote, 15min talk, 30min talk.
+- Is this your first time speaking at a conference?
+
+After we open the CFP, we begin communicating the CFP to our community members through our social media channels.
+
+Here is the stats of the CFP for the 2025 edition:
+
+![CFP stats](./images/cfp-stats.png)
+
+> It's important to note that we do not invite speakers and all speakers must submit their talk throw the CFP form.
+
+## 🤝 Content committee
+
+As we believe BlablaConf is a community conference and it's by developers for developers. We create a content committee of around 20 people to help us select the best talks.
```

Comment:
community driven*

Reply:
The by developers for developers term might be excluding some of our audience, such as designers PMs ... 
Maybe by IT enthusiast to Tech practitioners